tasks/firewall.yml

```diff
@@ -1,28 +1,29 @@
+---
 - name: Check for firewalld
-  shell: command -v firewall-cmd
+  command: which firewall-cmd
```

Comment:
I guess if we don't use the shell built in we don't need to use the shell module